### PR TITLE
feat(import script): recursiveDelete and promise.all

### PR DESCRIPTION
**Pull request by MaloPolese**

### CODE REVIEW

File: scripts/openplanner/import-speakers-sessions-schedule.ts [hunk 1]
```diff
@@ -137,55 +137,38 @@ export const importTeam = async (team: TeamMember[]) => {
...
-async function deleteQueryBatch(query: firestoreDep.Query, resolve: (value?: any) => any) {
   const snapshot = await query.get()
-
-  const batchSize = snapshot.size
-  if (batchSize === 0) {
-    // When there are no documents left, we are done
-    resolve()
-    return
+  for (const doc of snapshot.docs) {
+    await firestore.recursiveDelete(doc.ref);
```
Comment (HugoGresse) [inline scripts/openplanner/import-speakers-sessions-schedule.ts:142]: oh, this doesn't use batch :(

File: scripts/openplanner/import-speakers-sessions-schedule.ts [hunk 2]
```diff
@@ -137,55 +137,38 @@ export const importTeam = async (team: TeamMember[]) => {
...
     await cleanupScheduleSessionSpeakers()
-    await importSessions(data)
-    await importSpeakers(data)
-    await importSchedule(data)
-    await importSponsors(data)
+    await Promise.all([
+      importSessions(data),
+      importSpeakers(data),
+      importSchedule(data),
+      importSponsors(data)
```
Comment (HugoGresse) [inline scripts/openplanner/import-speakers-sessions-schedule.ts:170]: importing the team could also be done at the same time

File: scripts/openplanner/import-speakers-sessions-schedule.ts (hunk 1 above)
Comment (MaloPolese) [reply scripts/openplanner/import-speakers-sessions-schedule.ts:142]: Yes I spent a lot of time trying to do a batch transaction but in the end nothing was working.... 
From what I understood, this method uses a bulkWriter under the hood

Comment (HugoGresse) [reply scripts/openplanner/import-speakers-sessions-schedule.ts:142]: OKI! ❤️

**Review by HugoGresse — APPROVED** [review]

**Pull request merged**